lighthouse-core/test/report/report-generator-test.js

```diff
@@ -101,7 +101,6 @@ describe('ReportGenerator', () => {
       fs.writeFileSync(path, csvOutput);
 
       const lines = csvOutput.split('\n');
-      expect(lines).toHaveLength(145);
```

Comment:
can I sell you on `expect(lines.length).toBeGreaterThan(100)` or some such thing that ensures a bunch of audits made it through? :)

Reply:
I'm sold. As long as we don't have to update it all the time :)